### PR TITLE
Code coverage 2.2.0

### PR DESCRIPTION
**Pull request by zajck**
This PR already includes changes from #583, #589, #591 and #595 which should be reviewed first.

The first run with a new coverage version.

- Added missing tests
- Updated config
- Updated npm script
- Added coverage to CI

### CODE REVIEW

File: test/protocol/TwinHandlerTest.js [hunk 1]
```diff
@@ -377,12 +377,10 @@ describe("IBosonTwinHandler", function () {
...
         it("The twins region of protocol is paused", async function () {
           // Pause the twins region of the protocol
-          await pauseHandler
-            .connect(pauser)
-            .pause([PausableRegion.Offers, PausableRegion.Twins, PausableRegion.Bundles]);
+          await pauseHandler.connect(pauser).pause([PausableRegion.Twins]);
 
           // Attempt to Remove a twin, expecting revert
-          await expect(twinHandler.connect(assistant).removeTwin(twin.id)).to.revertedWith(RevertReasons.REGION_PAUSED);
+          await expect(twinHandler.connect(assistant).createTwin(twin)).to.revertedWith(RevertReasons.REGION_PAUSED);
```
Comment (anajuliabit) [inline test/protocol/TwinHandlerTest.js:383]: Why did you remove the `removeTwin` test?

File: test/protocol/TwinHandlerTest.js [hunk 2]
```diff
@@ -557,6 +555,7 @@ describe("IBosonTwinHandler", function () {
           await twinHandler.connect(assistant).createTwin(twin);
 
           // Create new twin with same token address
+          // twin.supplyAvailable = "2"
```
Comment (anajuliabit) [inline test/protocol/TwinHandlerTest.js:558]: 👀

**Review by anajuliabit — CHANGES_REQUESTED** [review]

File: test/protocol/TwinHandlerTest.js (hunk 1 above)
Comment (zajck) [reply test/protocol/TwinHandlerTest.js:383]: This line was wrong since it's inside `createTwin()` context. There were two tests for paused regions that used removeTwin and none that used `createTwin`.

Test for pausable region for `removeTwin` is in line 724.

File: test/protocol/TwinHandlerTest.js (hunk 2 above)
Comment (zajck) [reply test/protocol/TwinHandlerTest.js]: I guess I was testing my findings that led to #596 and I forgot to uncomment this.
I think it shouldn't affect the result, so I uncommented it.

**Review by mischat — APPROVED** [review]
LGTM

**Review by anajuliabit — APPROVED** [review]
